content/200-concepts/100-components/02-prisma-client/250-tracing.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: 'Tracing'
+metaTitle: 'Tracing (Preview)'
+metaDescription: 'Diagnose application performance with detailed traces of each query.'
+preview: true
+tocDepth: 3
+---
+
+<TopBlock>
+
+Prisma tracing logs
+Prisma tracing outputs
+
+Prisma tracing provides a detailed log of the activity that Prisma Client carries out, at a query level, including the time taken to execute each query. It helps you analyze your application's performance and identify bottlenecks. Tracing is fully compliant with [OpenTelemetry](https://opentelemetry.io/), so you can use it as part of your end-to-end application tracing path.
```

Comment:
```suggestion
Prisma tracing provides a detailed log of the activity that Prisma Client carries out, at an operation level, including the time taken to execute each query. It helps you analyze your application's performance and identify bottlenecks. Tracing is fully compliant with [OpenTelemetry](https://opentelemetry.io/), so you can use it as part of your end-to-end application tracing setup.
```

1. I think we should try and differentiate between operation and query. Prisma Client operations may lead to many SQL queries.